sources/packages/web/src/components/aest/institution/ApplicationOfferingChangeDetailsHeader.vue

```diff
@@ -112,7 +112,7 @@ export default defineComponent({
           assessedDateLabelValue.value =
             props.headerDetails.status ===
             ApplicationOfferingChangeRequestStatus.DeclinedByStudent
-              ? dateOnlyLongString(props.headerDetails.updatedDate)
+              ? dateOnlyLongString(props.headerDetails.studentActionDate)
```

Comment:
so, just wondering what will it show when a ministry user tries to view the page when the request is pending with the student or pending with the ministry? `studentActionDate` is only shown when the student declines the request. I believe `assessedDate` will only have value after the ministry action.

Reply:
Until there is a decline by student or a approve / decline by the ministry , the 2nd row is the header won't show. That is, in my understanding the expected behavior. Only the first row in the header shows as shown below:

<img width="1913" alt="image" src="https://github.com/bcgov/SIMS/assets/7859295/cc078368-47ba-456d-b310-75dfb3ace418">